src/components/Table/GridTable.tsx

```diff
@@ -294,7 +294,7 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   }, [columns, rows, sorting, sortState]);
 
   // Filter + flatten + component-ize the sorted rows.
-  let [headerRows, filteredRows]: [RowTuple<R>[], RowTuple<R>[]] = useMemo(() => {
+  let [headerRows, filteredRows, maxCardPadding]: [RowTuple<R>[], RowTuple<R>[], number | undefined] = useMemo(() => {
```

Comment:
It would be really great to move this into a function! Its a ~100 line useMemo :P